test/hotspot/jtreg/compiler/c2/cr7200264/TestIntVect.java

```diff
@@ -535,7 +543,15 @@ void test_divc(int[] a0, int[] a1) {
                    IRNode.VECTOR_SIZE + "min(max_int, max_long)", "> 0",
                    IRNode.SUB_VI,
                    IRNode.VECTOR_SIZE + "min(max_int, max_long)", "> 0" },
-        applyIfCPUFeatureOr = {"sse2", "true", "asimd", "true"})
+        applyIfCPUFeature = {"sse2", "true"},
+        applyIfAnd = {"UseAVX", ">= 2", "UseSSE", ">= 4"})
```

Comment:
Looks like you should just require `avx2`, instead of `sse2`. Then you can drop the `UseAvx` and `UseSSE` flag checks.

Reply:
If you set a lower AVX/SSE level, then the cpu-features are automatically removed. And if you have a cpu that does not have the relevant features, the AVX/SSE flags are automatically lowered. But in IR tests, we generally try to rely on the features as much as possible, and not the flags.

Reply:
OK, thanks, good to know. I'll make the update. @eme64: Do you want to create a new issue to investigate why it does not vectorize on aarch64? Or maybe that's expected?